src/components/pages/HomePage/HomePage.module.scss

```
@@ -1,77 +1,210 @@
+    > .slogan-subtitle {
+      font-weight: 300;
+      color: $typography-on-primary-light;
+      max-width: 196px;
+
+      @include mobile {
+        margin-bottom: 24px;
+
+        line-height: $typography-mobile-h5;
+        font-size: $typography-mobile-h5;
+      }
+
+      @include desktop {
+        line-height: $typography-desktop-h5;
+        font-size: $typography-desktop-h5;
+      }
+    }
 
-    @include mobile {
-      max-width: 320px;
-      line-height: $typography-mobile-h1;
-      font-size: $typography-mobile-h1;
+    > .action-button {
+      width: 100%;
+      display: block;
     }
 
-    @include desktop {
-      max-width: 648px;
-      line-height: $typography-desktop-h1;
-      font-size: $typography-desktop-h1;
+    > .learn-more-button {
+      display: block;
+      margin: 40px auto auto;
     }
+
   }
-  > .subtitle {
-    font-weight: 300;
-    color: white;
-    max-width: 196px;
-
-    @include mobile {
-      line-height: $typography-mobile-h5;
-      font-size: $typography-mobile-h5;
+
+  .second-section {
+    padding: 0;
+
+    > div {
+      padding-left: $content-horizontal-padding;
+      padding-right: $content-horizontal-padding;
     }
 
-    @include desktop {
-      line-height: $typography-desktop-h5;
-      font-size: $typography-desktop-h5;
+    > .methodology-list {
+      margin-top: 8px;
+
+      @include desktop {
+        @include content-spacing
+      }
+
+      .methodology-card {
+          margin-right: 60px;
+      }
+
     }
+
   }
-}
 
-.sectionHeadline {
-  display: block;
-  margin-bottom: 24px;
-}
+  .third-section {
+    padding-top: 24px;
 
-section {
-  margin: 48px 0;
-}
+    position: relative;
+    margin-top: 8px;
+    width: 100%;
 
-.case-study-list, .staff-member-list {
-  padding-top: 32px;
-}
+    overflow-y: hidden;
+    overflow-x: hidden;
 
-.case-study-card {
-  margin-bottom: 100px;
-}
+    .section-background {
+      $bg-height: 800px;
+
+      width: 240%;
+      height: $bg-height;
+
+      top: -11%;
+      left: -90%;
+
+      background: $gradient-section-bg;
+      position: absolute;
+      z-index: -10;
+      clip-path: ellipse(50% 40% at 50% 50%);
+      transform: rotate(8deg);
+
+      &::after {
+        content:"";
+        width: 100%;
+        height: 220px;
+        top: 550px;
+        background: $color-surface-light;
+        position: absolute;
+        left: 14%;
+        clip-path: ellipse(25% 55% at 50% 50%);
+      }
+    }
+
+    > .section-headline {
+      > h2, h3 {
+        color: $typography-on-primary-light;
+      }
+    }
+
+    > .action-button {
+      width: 100%;
+
+      margin-top: 4px;
+      margin-bottom: 24px;
+
+      background: $gradient-primary;
+      color: $typography-on-primary-light;
+
+      box-shadow: 0 8px 16px $color-shadow-elevated;
+
+      > .icon {
+        margin-right: 12px;
+        width: 24px;
+      }
+    }
+
+  }
+
+  .forth-section {
+    margin-top: 16px;
+    padding: 0;
+
+    > div {
```

Again replace by class.. However, we may refactor it later. But if we want to select all children, I suppose it's better to be agnostic of implementation element. Do you agree?

```suggestion
    > * {
```

---

If you want to apply this suggestion, remember there are other occurrencies of this.

---

I don't want to select all children! Just the div container I use to segregate content from the horizontal list.

---

I doubled checked and you were right! It the padding was also affecting the list. Since that was not my intention, I refactored it a little bit! :)